```
test: src/cpp/graph/datastructure/wrapper/2sat.hpp
src/cpp/graph/datastructure/wrapper/2sat.hpp: In member function ‘void Sat2::push(int, int)’:
src/cpp/graph/datastructure/wrapper/2sat.hpp:100:5: error: ‘assert’ was not declared in this scope
     assert(a != 0 && b != 0);
     ^~~~~~
src/cpp/graph/datastructure/wrapper/2sat.hpp:100:5: note: suggested alternative: ‘short’
     assert(a != 0 && b != 0);
     ^~~~~~
     short
failed!!
```

--- a/src/cpp/graph/datastructure/wrapper/2sat.hpp
+++ b/src/cpp/graph/datastructure/wrapper/2sat.hpp
@@ -21,6 +21,7 @@
 //
 // %require
 // ```
+#include <cassert>
 #include <vector>
 using namespace std;
 #include "src/cpp/graph/datastructure/dgraph.hpp"

